README.md

```diff
@@ -1,51 +1,46 @@
-# school_reports
+<img src="data/marketing/logo-header.png/">
 
-Dieses Projekt stellt eine einfache möglichkeit, professionelle und individuelle Wochenberichte für die IHK zu erstellen.
+<p align="right">
+  <a style="font-size: 7pt" href="https://www.flaticon.com/free-icons/school" title="school icons">School icons created by Freepik - Flaticon</a>
+</p>
 
-## Disclaimer
+---
+
+**school reports** is a simple way to create, store and customize your reports.
 
-* Jeder ist für den Inhalt, die Formatierung und Abgabe der Berichte selbst verantwortlich
-* Wenn Änderungen gewünscht sind oder Bugs auftreten einen Git Issue anlegen!
-* Bei jedem pull wird report.tex und config.py überschrieben. Bedenkt das!
+## Disclaimer
+  - Every student is responsible for their reports. No developer takes any blame if your reports do not fulfill the requirements for your school or institute you may take full responsibilty for this.
```

Comment:
seems a bit to german lol

Reply:
write something like "Every student is responsible for their own reports. No developer will take any blame if your reports do not fulfill the requirements of your school or institute. You bear full responsibility for this."

Reply:
Yeah I changed that xD
I took the sentence you wrote since it sounded good.